From 7f958fd684018cee68c195590cf26d5122eabd0d Mon Sep 17 00:00:00 2001
From: Meir Levi <mlevi4@marvell.com>
Date: Thu, 24 Dec 2020 10:43:38 -0500
Subject: [PATCH 1068/1921] Documentation :devicetree: bindings: marvell UIO
 XOR documentation

Marvell UIO for XOR driver documentation

Signed-off-by: Meir Levi <mlevi4@marvell.com>
Change-Id: I05a3fd4b972ee24763e10d6d7feab91a0e44cf96
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/42778
Tested-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 .../devicetree/bindings/marvell-uio-xor.txt          | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/marvell-uio-xor.txt

diff --git a/Documentation/devicetree/bindings/marvell-uio-xor.txt b/Documentation/devicetree/bindings/marvell-uio-xor.txt
new file mode 100644
index 000000000000..a41c3e859b96
--- /dev/null
+++ b/Documentation/devicetree/bindings/marvell-uio-xor.txt
@@ -0,0 +1,12 @@
+* Marvell UIO for XOR driver
+
+Required properties:
+
+- compatible: should be "marvell,uio-xor-v2".
+- xor_access: phandler to XOR driver
+
+Example:
+P110_LABEL(uio_xor1) {
+	compatible = "marvell,uio-xor-v2";
+	xor_access = <&CP110_LABEL(xor1)>;
+};
-- 
2.31.1

